pkg/kubeapiserver/options/authentication.go

```diff
@@ -304,6 +305,10 @@ func (s *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 		fs.DurationVar(&s.ServiceAccounts.MaxExpiration, "service-account-max-token-expiration", s.ServiceAccounts.MaxExpiration, ""+
 			"The maximum validity duration of a token created by the service account token issuer. If an otherwise valid "+
 			"TokenRequest with a validity duration larger than this value is requested, a token will be issued with a validity duration of this value.")
+
+		fs.BoolVar(&s.ServiceAccounts.ExtendExpiry, "service-account-extend-token-expiry", s.ServiceAccounts.ExtendExpiry, ""+
```

Comment:
nit, use `expiration` to match the other flag

Reply:
Done